.github/workflows/post-merge-tests.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     - name: Build Cantera
       run: python3 `which scons` build env_vars=all
         CXX=clang++-14 CC=clang-14 f90_interface=n extra_lib_dirs=/usr/lib/llvm/lib
-        -j4 debug=n --debug=time logging=debug
+        -j4 debug=n --debug=time logging=debug python_package=y
```

Comment:
I've been wondering if we should update the default to build the Python package unless it is explicitly disabled, so that missing a Python dependency results in a build failure unless you set `python_package=n`.

Reply:
I'm for it! Do you want to make an enhancements issue so we can at least try to document some tradeoffs?